server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/tservers.js

```diff
@@ -92,10 +88,13 @@ $(document).ready(function() {
         // reset background of each row
         $(row).css('background-color', '');
 
-        // return if current rows tserver is not recovering
+        // return if the current rows' tserver is not recovering
```

Comment:
I am bad with grammer but I think `row's` is what you want here...
"The general rule is that the possessive of a singular noun is formed by adding an apostrophe and s, whether the singular noun ends in s or not." i.e. `the child’s toy`